extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -376,14 +376,14 @@ export class AmpIframe extends AMP.BaseElement {
    */
   updateSize_(height, width) {
     if (!this.isResizable_) {
-      user.warn(TAG_,
+      user.error(TAG_,
```

Comment:
@cramforce do you want these to be error? for the 100px i think it is violating a particular spec. (that we will add to the MD file.) The rest of the cases are most likely from ads that dont know what type their container is (non resizable) or incorrect values in the resize call , i think they should be warnings. wdyt?